libs/ledger-live-common/src/account/formatters.ts

```diff
@@ -93,11 +95,9 @@ const cliFormat = (account, level?: string) => {
     account.balance,
     getAccountCurrency(account).units[0],
   );
-  const family = byFamily[account.currency.family];
 
-  if (family && family.formatAccountSpecifics) {
-    str += family.formatAccountSpecifics(account);
-    str += "\n";
+  if (getAccountBridge(account).formatAccountSpecifics) {
+    str += getAccountBridge(account).formatAccountSpecifics?.(account) + "\n";
```

Comment:
You can get correct typings here I think with a constant
```suggestion
  const formatAccountSpecifics = getAccountBridge(account).formatAccountSpecifics
  if (formatAccountSpecifics) {
    str += formatAccountSpecifics(account) + "\n";
```

Another solution could be to keep the accountBridge in a const (not sure if it works as expected with typescript like the previous one)
```suggestion
  const accountBridge = getAccountBridge(account);
  if (accountBridge.formatAccountSpecifics) {
    str += accountBridge.formatAccountSpecifics(account) + "\n";
```